copier/main.py

```diff
@@ -130,6 +145,9 @@ class Worker:
             When `True`, disable all output.
 
             See [quiet][].
+
+        conflict:
+            One of "rej", "inline"
```

Comment:
To keep it homogeneous and warn about this new algorithm status:

```suggestion
            One of "rej" (default), "inline" (still experimental).
```